app/src/main/java/com/kickstarter/ui/activities/ProjectPageActivity.kt

```diff
@@ -0,0 +1,792 @@
+import android.animation.AnimatorSet
+import android.animation.ObjectAnimator
+import android.content.Intent
+import android.graphics.Rect
+import android.os.Bundle
+import android.util.Pair
+import android.view.MotionEvent
+import android.view.View
+import android.view.ViewTreeObserver
+import android.widget.EditText
+import android.widget.LinearLayout
+import androidx.annotation.MenuRes
+import androidx.annotation.StringRes
+import androidx.appcompat.app.AlertDialog
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.FragmentManager
+import com.google.firebase.crashlytics.FirebaseCrashlytics
+import com.kickstarter.R
+import com.kickstarter.databinding.ActivityProjectBinding
+import com.kickstarter.libs.ActivityRequestCodes
+import com.kickstarter.libs.BaseActivity
+import com.kickstarter.libs.BaseFragment
+import com.kickstarter.libs.Either
+import com.kickstarter.libs.KSString
+import com.kickstarter.libs.MessagePreviousScreenType
+import com.kickstarter.libs.qualifiers.RequiresActivityViewModel
+import com.kickstarter.libs.rx.transformers.Transformers
+import com.kickstarter.libs.utils.ApplicationUtils
+import com.kickstarter.libs.utils.TransitionUtils
+import com.kickstarter.libs.utils.ViewUtils
+import com.kickstarter.libs.utils.extensions.toVisibility
+import com.kickstarter.models.Project
+import com.kickstarter.models.StoredCard
+import com.kickstarter.ui.IntentKey
+import com.kickstarter.ui.adapters.ProjectAdapter
+import com.kickstarter.ui.data.CheckoutData
+import com.kickstarter.ui.data.LoginReason
+import com.kickstarter.ui.data.PledgeData
+import com.kickstarter.ui.data.PledgeReason
+import com.kickstarter.ui.data.ProjectData
+import com.kickstarter.ui.extensions.hideKeyboard
+import com.kickstarter.ui.extensions.showSnackbar
+import com.kickstarter.ui.fragments.BackingFragment
+import com.kickstarter.ui.fragments.CancelPledgeFragment
+import com.kickstarter.ui.fragments.NewCardFragment
+import com.kickstarter.ui.fragments.PledgeFragment
+import com.kickstarter.ui.fragments.RewardsFragment
+import com.kickstarter.viewmodels.ProjectViewModel
+import com.stripe.android.view.CardInputWidget
+import rx.android.schedulers.AndroidSchedulers
+
+@RequiresActivityViewModel(ProjectViewModel.ViewModel::class)
+class ProjectPageActivity :
+    BaseActivity<ProjectViewModel.ViewModel>(),
+    CancelPledgeFragment.CancelPledgeDelegate,
+    NewCardFragment.OnCardSavedListener,
+    PledgeFragment.PledgeDelegate,
+    BackingFragment.BackingDelegate {
+    private lateinit var adapter: ProjectAdapter
+    private lateinit var ksString: KSString
+
+    private val projectShareLabelString = R.string.project_accessibility_button_share_label
+    private val projectShareCopyString = R.string.project_share_twitter_message
+    private val projectStarConfirmationString = R.string.project_star_confirmation
+
+    private val animDuration = 200L
+    private lateinit var binding: ActivityProjectBinding
```

Comment:
You should be loading anothe XML file here, therefor another Binding, without the recyclerView